curl-websocket/curl-websocket.c

```diff
@@ -775,6 +787,11 @@ static size_t _cws_process_frame(struct cws_data *priv, const char *buffer, size
 static size_t _cws_receive_data(const char *buffer, size_t count, size_t nitems, void *data) {
     struct cws_data *priv = data;
     size_t len = count * nitems;
+
+    if (priv->redirection) {
+        return count * nitems;
```

Comment:
this is `len` computed above

Reply:
good catch - I missed the line 2 lines up :)